custom_dc/README.md

```diff
@@ -189,6 +189,43 @@ Console](https://console.cloud.google.com/storage/browser), create a new bucket
 or pick an existing bucket and upload the data CSV files there. It's recommended
 to create intermediate folders for the files for easier management.
 
+### (Optional) Setup Redis
+
+Configure redis as website caching layer to improve performance. Open the
+[Google Cloud Redis Memorystore Console](https://console.cloud.google.com/memorystore/redis/instances)
+and configure a new redis instance. Example:
+
```

Comment:
Since the vpc network was created using gloud command, maybe also do it here? Or even wrap all commands in a script?

Reply:
Added gcloud commands inline to keep it consistent with the rest of the readme